prototypes/templates.lua

```diff
@@ -1,40 +1,68 @@
+-- Converts hex code values to rgb values, alpha is an optional parameter
+local function tint_from_hex(hex, alpha)
```

Comment:
Could you use [util.color](https://github.com/wube/factorio-data/blob/master/core/lualib/util.lua#L65-L78)?